src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.ts

```diff
@@ -106,11 +107,21 @@ export class InstructorHelpStudentsSectionComponent extends InstructorHelpSectio
     ],
   }];
 
+  @Input() collapseStudentEditDetailsInChild: Subject<boolean> = new Subject<boolean>();
+  isEditDetailsCollapsed: boolean = false;
+  isViewProfileCollapsed: boolean = false;
+  isViewAllRespCollapsed: boolean = false;
+  isStudSearchCollapsed: boolean = false;
+  isStudEmailCollapsed: boolean = false;
+  isGoogleAccCollapsed: boolean = false;
+  isChangeGoogleIDCollapsed: boolean = false;
```

Comment:
```suggestion
  isChangeGoogleIdCollapsed: boolean = false;
```

Just a nit to adhere to camelCase

Reply:
Done.